assets/multiqc_config.yml

```diff
@@ -1,12 +1,35 @@
 report_comment: >
   This report has been generated by the <a href="https://github.com/Arcadia-Science/seqqc" target="_blank">Arcadia-Science/seqqc</a>
-  analysis pipeline.
+  analysis pipeline. The purpose of this pipeline is to rapidly assess the quality of new sequencing data so that you can feel 
+  confident depositing it into an <a href="https://www.insdc.org/" target="_blank">INSDC database</a> like the 
+  <a href="https://www.ebi.ac.uk/ena/browser/home" target="_blank">European Nucleotide Archive</a> rapidly after data generation.
+  In typical academic settings, data would be deposited in a an INSDC database at time of publication, so any quality issues with
+  the data itself would be caught during the analysis. The seqqc pipeline was designed to perform minimum quality control reporting
+  so that data could be posted more closely at time of generation, but also so that quality problems would still be caught prior to 
+  deposition. The pipeline is designed to report technical sequencing issues via FastQC, sample similarity with sourmash compare, 
+  and potential contamination with sourmash gather. Because this pipeline is designed to run on any type of raw sequencing data, it 
+  is purely descriptive and you will need to rely on a combination of your domain specific expertise, the section descriptions, 
+  and the results to determine whether there is a quality issue with your sequencing data or not. For information about how to 
+  interpret these results, please consult each section of this document.
 report_section_order:
-  "Arcadia-Science-seqqc-methods-description":
+  fastqc:
```

Comment:
oh cool is this how you order the reports for each module?

Reply:
ya!